extensions/scarb-doc/src/docs_generation/markdown/traits.rs

```diff
@@ -215,3 +204,27 @@ fn generate_markdown_from_item_data(doc_item: &dyn DocItem, header_level: usize)
 
     markdown
 }
+
+pub trait TopLevelMarkdownItemsSummary<'a> {
+    const ITEMS_SUMMARY_FILENAME: &'a str;
+}
```

Comment:
The purpose of this trait is to avoid hardcoding the names multiple times (just hardcode them here)